src/richie/apps/core/tests/utils.py

```diff
@@ -5,6 +5,47 @@
 import os
 import random
 
+from cms.test_utils.testcases import CMSTestCase
+
+
+class CMSPluginTestCase(CMSTestCase):
+    """
+    Enriched CMS test case object to include useful stuff about plugin
+    rendering.
+    """
+
+    def get_practical_plugin_context(self, extra_context=None):
+        """
+        Build a template context with dummy request object and
+        instanciated content renderer suitable to perform full rendering of
+        any plugin.
+
+        Note:
+            CMSTestCase use a dummy AnonymousUser on default behavior you may
+            override with a custom user if you an ``user`` attribute to your
```

Comment:
you wrote: "if you ... an user" -> missing a word and "an" should be "a".

Reply:
Note: We don't want to use the self.user from TestCase. We should use UserFactory inside each test to be explicit and self referencing.